### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by The7thSun** [community/community#47264] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hello all, I'm looking for some help rendering my page as I'm very new to GitHub and I'm not sure what I'm doing wrong. I created my repository and also added all my files, but my render continues to fail. 

Here is the link to my page 
https://the7thsun.github.io/1The7thSun.github.io/

**Comment by radar-blip** [comment, marked as answer]
This link here (in index.html):
`<img src="../savon-stephens-resume/images/savons-resume-picture.jpg " alt="picture of savon stephens">`
should be: 
`<img src="images/savons-resume-picture.jpg " alt="picture of savon stephens">`

Also, try changing this line `<section style="margin-top: -25%">` to something like 2% and see if it helps.

**Comment by The7thSun** [comment]
That helped a ton thank you so much!!

**Reply by radar-blip** [reply]
You're welcome! 👍🏻

**Comment by OlaAnobi** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> Hello all, I'm looking for some help rendering my page as I'm very new to GitHub and I'm not sure what I'm doing wrong. I created my repository and also added all my files, but my render continues to fail.
> 
> Here is the link to my page https://the7thsun.github.io/1The7thSun.github.io/
